# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/moz.build
# Commit: 77b0f5226ed8
# Full Hash: 77b0f5226ed854c6b8d1b8d13f3106c12ad0701f
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-30 04:06:43
# Regressor Bug: 1831451
# File Overlap Count: 2
# Description:
#   Bug 1831451 - Add a blank VideoDecoder interface r=webidl,padenot,edgar,smaug,saschanaz
#   
#   This patch introduces a VideoDecoder interface generated by running
#   `./mach webidl-example VideoDecoder` with necessary changes to make it
#   buildable.
# ==============================================================================

diff -r fb3b06cc9188 -r 77b0f5226ed8 dom/webidl/moz.build
--- a/dom/webidl/moz.build	Thu Jun 29 22:25:47 2023 +0000
+++ b/dom/webidl/moz.build	Thu Jun 29 22:33:11 2023 +0000
@@ -977,6 +977,7 @@
     "URLSearchParams.webidl",
     "ValidityState.webidl",
     "VideoColorSpace.webidl",
+    "VideoDecoder.webidl",
     "VideoFrame.webidl",
     "VideoPlaybackQuality.webidl",
     "VideoTrack.webidl",
